src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumModule.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.bridge.sdk;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.Deserializers;
+import com.fasterxml.jackson.databind.deser.Deserializers.Base;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+@SuppressWarnings("serial")
+class LowercaseEnumModule extends SimpleModule {
+
+    public LowercaseEnumModule() {
+        super("bridge-enum-module", new Version(1, 0, 0, "", "org.sagebionetworks.bridge", "sdk"));
+        addSerializer(Enum.class, new LowercaseEnumSerializer());
+    }
+    
+    @Override
+    public void setupModule(SetupContext context) {
+        super.setupModule(context);
+        Base deser = new Deserializers.Base() {
+            @SuppressWarnings("unchecked")
+            @Override
+            public JsonDeserializer<?> findEnumDeserializer(Class<?> type, DeserializationConfig config,
+                    BeanDescription beanDesc) throws JsonMappingException {
+                return new LowercaseEnumDeserializer((Class<Enum<?>>) type);
+            }
```

Comment:
We cache the object mapper we're constructing, (we create it statically when Utilities is loaded), and it is thread-safe to re-use. According to the docs, this setup method is called once when the module is registered with the mapper.